modules/complete-assisted-installer-oci.adoc

```diff
@@ -1,33 +1,37 @@
 // Module included in the following assemblies:
 //
-// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+// * installing/installing_oci/installing-oci-assisted-installer.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="complete-assisted-installer-oci_{context}"]
 = Completing the remaining Assisted Installer steps
 
-After you provision {oci-first} resources and upload {product-title} custom manifest configuration files to {oci}, you must complete the remaining cluster installation steps on the Assisted Installer before you can create an instance {oci}.
+After you provision {oci-first} resources and upload {product-title} custom manifest configuration files to {oci}, you must complete the remaining cluster installation steps on the {ai-full} before you can create an instance {oci}.
 
 .Prerequisites
 
-* You created a resource stack on {oci}, and the stack includes the custom manifest configuration files and {oci} Resource Manager configuration resources.
+* You created a resource stack on {oci} that includes the custom manifest configuration files and {oci} Resource Manager configuration resources. See the "Downloading manifest files and deployment resources" section.
 
 .Procedure
 
-. From the link:https://console.redhat.com/[Red Hat Hybrid Cloud Console] web console, go to the **Host discovery** step. Under the **Role** column, assign a node role, `Control plane node` or `Worker`, for each targeted hostname.
+. From the link:https://console.redhat.com/[Red Hat Hybrid Cloud Console] web console, go to the *Host discovery* page.
+
+.Under the *Role* column, select either `Control plane node` or `Worker` for each targeted hostname.
 +
 [IMPORTANT]
 ====
 Before, you can continue to the next steps, wait for each node to reach the `Ready` status.
 ====
 
-. Accept the default settings for the **Storage** and **Networking** steps. Click the **Next** button to go to the **Custom manifests** step. 
+. Accept the default settings for the *Storage* and *Networking* steps, and then Click *Next*.
```

Comment:
typo: "Click"

Reply:
Thanks. For a follow-up PR. I need to address merge conflicts on 4.14-4.16